src/main/java/core/basesyntax/service/OperationStrategy.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax.service;
+
+import core.basesyntax.service.operations.Expense;
+import core.basesyntax.service.operations.Income;
+import core.basesyntax.service.operations.OperationService;
+import java.util.HashMap;
+import java.util.Map;
+
+public class OperationStrategy {
+    private static Map<Operation, OperationService> events = new HashMap<>(4);
+
+    static {
+        events.put(Operation.RETURN, new Expense());
```

Comment:
r (RETURN) it is addition operation, you should paste here **Income()** strategy.

r - return, means someone who have bought the fruits now returns them back